VaticanBAV/et/BAVet117.xml

```diff
@@ -738,72 +728,115 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                            ተአምር፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ወሰቀሉ፡ ምስለ፡ <del>እ</del>እግዚእነ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ፪ፈያተ፡ ለ፩ስሙ፡ ጥጦስ፡ ሰቀልዎ፡ በየማኑ፡ ወስመ፡ ካልኡ፡ ዳክርስ። ወለዝ፡ ሰቀልዎ፡ በፀጋሙ፡
+                            ተአምር፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ወሰቀሉ፡ ምስለ፡ <del>እ</del>እግዚእነ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ፪ፈያተ፡ ለ፩ስሙ፡ ጥጦስ፡ ሰቀልዎ፡ በየማኑ፡ ወስመ፡ ካልኡ፡ ዳክርስ። ወለዝ፡ ሰቀልዎ፡ በፀጋሙ፡
                           </incipit>
                           <explicit xml:lang="gez">
-                            ዮም፡ ትሄሉ፡ ምስሌየ፡ ህየ፡ ወትቀድሞ፡ ለአዳም፡ በዊኦ፡ ውስተ፡ ገነት፨ ወእምዝ፡ ይቤሎ፡ ኦጥጦስ፡ እስመ፡ ተፈጸመ፡ ዮም፡ ነገርየ፡ ዘእቤለከ፡ በውስተ፡ ፍኖተ፡ ግብፅ፡ ሣህሉ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
+                            ዮም፡ ትሄሉ፡ ምስሌየ፡ ህየ፡ ወትቀድሞ፡ ለአዳም፡ በዊኦ፡ ውስተ፡ ገነት፨ ወእምዝ፡ ይቤሎ፡ ኦጥጦስ፡ እስመ፡ ተፈጸመ፡ ዮም፡ ነገርየ፡ ዘእቤለከ፡ በውስተ፡ ፍኖተ፡ ግብፅ፡ ሣህሉ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
                           </explicit>
                         </msItem>
 
                         <msItem xml:id="p2_i1.30">
                           <locus from="89v" to="91r"/>
-                          <title type="complete" ref="LIT4661Miracle">On the Death of Jesus</title>
+                          <title type="complete" ref="LIT4661Miracle">A Miracle Concerning the Death of Jesus</title>
                           <incipit xml:lang="gez">
-                            ተአምር፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ወእምድኅረ፡ ይቤሎሙ፡ እግዚእነ፡ ለወዓልት፡ እለ፡ <supplied reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran">፡</supplied> የዓቅብዎ፡ ፀማዕኩ፨
+                            ተአምር፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ወእምድኅረ፡ ይቤሎሙ፡ እግዚእነ፡ ለወዓልት፡ እለ፡ <supplied reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran">፡</supplied> የዓቅብዎ፡ ፀማዕኩ፨
                           </incipit>
                           <explicit xml:lang="gez">
-                            ወጸልመት፡ ምድር፡ እምጊዜ፡ ፮ሰዓት፡ ዘመዓልት፡ እስከ፡ <choice><sic>ተ</sic></choice>፱ሰ<del>ዓ</del>ት፨ በረከተ፡ ምሕረቱ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
+                            ወጸልመት፡ ምድር፡ እምጊዜ፡ ፮ሰዓት፡ ዘመዓልት፡ እስከ፡ <choice><sic>ተ</sic></choice>፱ሰ<del>ዓ</del>ት፨ በረከተ፡ ምሕረቱ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
                           </explicit>
                         </msItem>
 
                       </msItem>
                       </msContents>
+
+                      <physDesc>
+                        <objectDesc form="Codex">
+                          <supportDesc>
+                            <support>
+                              <material key="parchment"/>
+                            </support>
+                            <collation>
```

Comment:
I would not split collation (it belongs to the entire unit of circulation as it is now)
But I am missing the extent (number of folia, blank folia) and measurements / layout if known

Reply:
Collation can also be split between the units of production, both are fine